src/views/ProductShell.vue

```diff
@@ -1,9 +1,9 @@
 <template>
-  <div class="container flex pb-0 product fixed-position">
+  <div class="product-shell product fixed-position container">
```

Comment:
I made a `.responsive-container` class in my PR that should replace container, as that is a tailwind utility class also.

Reply:
Yeah, I'll wait to merge mine until yours is good 2 go.